src/Node/Expression/Filter/RawFilter.php

```diff
@@ -11,20 +11,23 @@
 
 namespace Twig\Node\Expression\Filter;
 
+use Twig\Attribute\FirstClassTwigCallableReady;
 use Twig\Compiler;
 use Twig\Node\Expression\ConstantExpression;
 use Twig\Node\Expression\FilterExpression;
 use Twig\Node\Node;
+use Twig\TwigFilter;
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
 class RawFilter extends FilterExpression
 {
-    public function __construct(Node $node, ?ConstantExpression $filterName = null, ?Node $arguments = null, int $lineno = 0, ?string $tag = null)
+    #[FirstClassTwigCallableReady]
+    public function __construct(Node $node, TwigFilter|ConstantExpression|null $filterName = null, ?Node $arguments = null, int $lineno = 0, ?string $tag = null)
```

Comment:
Should `$filterName` be renamed $filter ?

Reply:
Indeed, done